packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Microsoft.Generator.CSharp.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I see these often in the PRs, do we know what is causing this?  Is it a BOM setting on one of our machines?